_includes/manuals/2.2/1.2_release_notes.md

```diff
@@ -16,14 +12,252 @@ aren't familiar with all the methods Foreman provides. Documentation is availabl
+
+#### foreman-installer script
+* make it possible to brand katello-certs-check ([#30251](https://projects.theforeman.org/issues/30251))
+* Remove MongoDB Storage Engine Migration Hook ([#30006](https://projects.theforeman.org/issues/30006))
+* Remove the dependency on foreman-maintain ([#29963](https://projects.theforeman.org/issues/29963))
+* Drop --upgrade flag from katello hooks ([#29296](https://projects.theforeman.org/issues/29296))
+
+#### foreman-tasks
+* During smart proxy sync display the filter is set to "resource_id = 3 AND resource_type = SmartProxy", the tasks are displayed twice and Foreman Task id is set to "zero" ([#30376](https://projects.theforeman.org/issues/30376))
+
+*A full list of changes in 2.2.0 is available via [Redmine](https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f[]=cf_12&op[cf_12]=%3D&v[cf_12][]=1234)*
 
 ### Deprecations
 
 ### Upgrade warnings
 
-* Trends and statistics are now moved to separate plugin. See [manual](/plugins/foreman_statistics) of the new Statistics plugin if you wish continue using it.
+* The `--upgrade` installer flag has been removed. When running the installer as part of the upgrade procedure it is not required.
+* Trends and statistics are now moved to a separate plugin. See the [manual](/plugins/foreman_statistics) of the new Statistics plugin if you wish continue using them.
 * TFTP integration is now [disabled by default](https://community.theforeman.org/t/disable-foreman-proxy-tftp-support-by-default/18892). Upgrades are unaffected. To enable TFTP integration, `--foreman-proxy-tftp true` must be passed to the installer command.
+* Puppet run button has been removed from the host detail page. To trigger one-time puppet agent runs, please use the Remote Execution plugin instead. ([More info](https://community.theforeman.org/t/dropping-run-puppet-button-with-a-replacement-using-remote-execution-plugin/12781/25))
+* Previously the default installation set the `destroy_vm_on_host_delete` setting to `true`, meaning associated VMs were automatically destroyed upon host deletion in Foreman. This caused a lot of pain for users who did not expect such behavior, and mistakenly destroyed production machines when attempting to remove them from Foreman. The default behavior has been changed and VMs will now remain running on the hypervisor or in the cloud. Notice that this can lead to unexpected payments in case you are paying for these VMs. If you prefer to destroy VMs automatically upon host deletion, switch this setting back to `true`.
+* The template helper named `foreman_url` now requires an argument which represents which template kind to render. Previously, no argument was needed to render `provision` template kind. All default templates were updated, for custom templates please rewrite `foreman_url` to `foreman_url('provision')`.
 
 ### Contributors
```

Comment:
Looks like you didn't run this part yet

Reply:
right, i need the ga tag for the contributor script to gather the names. I do that as part of the GA update.